jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/PluginConfigurationProcessor.java

```diff
@@ -251,11 +252,11 @@ static PluginConfigurationProcessor processCommonConfiguration(
             .setProgramArguments(rawConfiguration.getProgramArguments().orElse(null))
             .setEnvironment(rawConfiguration.getEnvironment())
             .setExposedPorts(ExposedPortsParser.parse(rawConfiguration.getPorts()))
-            .setVolumes(getVolumesList(rawConfiguration))
+            .setVolumes(new HashSet<>(getVolumesList(rawConfiguration)))
```

Comment:
Why do you need `new HashSet()`, especially given that you didn't need it for `setExposedPorts()`?